conditional_game.py

```diff
@@ -49,28 +51,50 @@
 
 	# Part 2: Next, we need to randomly select a response from 20 options.
 
-	# Randomly select a number from 0 - 19 
+	# Randomly select a number from 0 - 19       
 	# Use that to select from the following responses:
-		# 0 - It is certain.
-		# 1 - It is decidedly so.
-		# 2 - Without a doubt.
-		# 3 - Yes - definitely.
-		# 4 - You may rely on it.
-		# 5 - As I see it, yes.
-		# 6 - Most likely.
-		# 7 - Outlook good.
-		# 8 - Yes.
-		# 9 - Signs point to yes.
-		# 10 - Reply hazy, try again.
-		# 11 - Ask again later.
-		# 12 - Better not tell you now.
-		# 13 - Cannot predict now.
-		# 14 - Concentrate and ask again.
-		# 15 - Don't count on it.
-		# 16 - My reply is no.
-		# 17 - My sources say no.
-		# 18 - Outlook not so good.
-		# 19 - Very doubtful.
+	select=random.randrange(1,20)
+	if  select=0
+	print(" It is certain.")
```

Comment:
1. Since you are trying to determine if `select` _is equal to_ `0` you should use the comparison operator `==` instead of the assignment operator (`=`). 
2. Add 1 space between your keywords, values, and operators. The same is true for the `elif` statements below. 
3. There should be a colon (`:`) at the end of every `if`, `elif `or `else` statement. 
4. The code inside the blocks (after the `:`) should be indented. 

```suggestion
if select == 0:
    print("It is certain.")
```